api/cards/getDeck.ts

```diff
@@ -1,11 +1,38 @@
 import { Request, Response } from "express";
+import { ICardDefinition } from "~/typings/ICardDefinition";
 import { filterKnownCards, getAllCards } from "~/utils/cards";
 import { generateDeck } from "~/utils/decks";
 
-export default (_: Request, response: Response) =>
-  getAllCards()
+export default (request: Request, response: Response) => {
+  const params = request.query;
```

Comment:
Also I'm interested in some alternatives to this approach (to get tags from url query params)

Reply:
what's wrong with this one? the only -  it's better to add param validation to endpoint